utilities/habanero/mod_env_setup.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+load module anaconda
```

Comment:
isn't it `module load anaconda`?

Reply:
You are right. Let me fix that real quick.